tests/api/test_item_loan_request.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test loans item permissions."""
+
+from __future__ import absolute_import, print_function
+
+import json
+
+import pytest
+from flask import url_for
+
+from invenio_accounts.models import User
+from invenio_accounts.testutils import login_user_via_session
+
+NEW_LOAN = {
```

Comment:
any particular reason not to be a fixture?Just wondering..

Reply:
I normally create a fixture when they are common to several tests and modules. In this case, I am not sure we are gonna use it for other tests, it may vary...  I just did not want to have A LOT of fixtures...